### ISSUE TITLE
[html5 target] mouseEnabled not working

### ISSUE THREAD

**Issue by Celsyum** [openfl/openfl#1777]
one clickable sprite is not working under another , which has mouseEnabled=false applied.
Not works on openfl 6.5.1 version, html target. Should useHandCursor  parameter work too ?
Here is code:
```
package;

import openfl.events.MouseEvent;
import openfl.display.Sprite;

class Main extends Sprite 
{

	public function new() 
	{
		super();
		
		var sp:Sprite = getSprite(100, 50, 0, 0);
		sp.addEventListener(MouseEvent.CLICK, onClick);
		sp.useHandCursor = true; // not working too ?
		addChild(sp);
		
		sp = getSprite(70, 50, 20, 10);
		addChild(sp);
		sp.mouseEnabled = false; // not working anymore on 6.5.1		
	}
	
	function getSprite(w:Float, h:Float, x:Float, y:Float):Sprite
	{
		var sp:Sprite = new Sprite();
		sp.graphics.clear();
		sp.graphics.beginFill(0xFBFBFB, 0.2);
		sp.graphics.lineStyle(1, 0, 0.4);
		sp.graphics.drawRect(0, 0, w, h);
		sp.x = x;
		sp.y = y;
		return sp;
	}
	
	function onClick(e:MouseEvent):Void 
	{
		trace("clickable", cast(e.currentTarget, Sprite).name);
	}

}
```

**Comment by Celsyum** [comment]
this issue persists on 6.5.3 version

**Comment by miltoncandelero** [comment]
Can confirm.

Anything above OpenFL 6.5.0 will break the behavior of "mouseEnabled" on html5 targets.

(Current version at the moment of writing: 6.5.3)

**Comment by jgranick** [comment]
We improved `sprite.graphics` hit testing to return `true`, but not return itself as the hit target if it's `mouseEnabled` property is false. This makes it's parent return as the hit target. In this test you made, the correct behavior was for the top `Sprite` to hit test, but for the hit test process to continue, until it found the bottom `Sprite`, and returned that as the hit target.

I've made a fix that resolves this sample. By the way, `sp.useHandCursor` doesn't enable a cursor on the Flash target in your test, but adding `sp.buttonMode = true;` as well does :smile:
